fivox/CMakeLists.txt

```diff
@@ -24,7 +24,6 @@ set(FIVOX_PUBLIC_HEADERS
 )
 
 # Try cloning lfpFivox into the 'fivox' folder
-set(FIVOXLFP_DIR ${PROJECT_SOURCE_DIR}/fivox/lfp)
```

Comment:
FIVOXLFP_DIR is also used in tests/CMakeLists.txt. This line was moved in the top CMakeLists.txt for that reason.